index.html

```diff
@@ -8,25 +8,29 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <nav class="flex-div">
-        <div class="nav-left">
-            <h3 class="logo">My Logo</h3>
-        </div>
-        <div class="nav-right">
-            <img src="src/img/Union.png" alt="des-menu" class="menu">
-        </div>
-    </nav>
+    <header class="flex-div">
+        <a><h3 class="logo">My Logo</h3></a>
+        <nav>
+            <div class="nav-right">
+                <img src="src/img/Union.png" alt="des-menu" class="menu">
+            </div>
+        </nav>
+    </header>
     <section class="main-container">
         <div class="greeting-text">
-            <h2 class="name">I'm Jhonatan.<br>Glad to see you!</h2>
+            <h1 class="name">I'm Jhonatan.<br>Glad to see you!</h1>
             <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
         </div>
         <div class="connect">
             <p>LET'S CONNECT</p>
         </div>
-        <div class="media-icon">
-            <img src="src/img/social media icons.png" alt="">
-        </div>
+        <ul class="media-icon">
+            <a href="" class="icon1"><img src="src/img/icons/Vector.png" alt="Twitter"></a>
+            <a href="" class="icon2"><img src="src/img/icons/Linkedin icon.png" alt="Linkedin"></a>
+            <a href="" class="icon3"><img src="src/img/icons/Vector (1).png" alt="icon3"></a>
+            <a href="" class="icon4"><img src="src/img/icons/Vector (2).png" alt="icon4"></a>
+            <a href="" class="icon5"><img src="src/img/icons/Vector (3).png" alt="icon5"></a>
+        </ul>
```

Comment:
- [ ] Because this is a list of elements, please ensure that each `a` tag is wrapped inside a `li` tag, as requested by the previous reviewer. :ok_hand: